### ISSUE TITLE
Reduce or eliminate manual data entry in sample labeling

### ISSUE THREAD

**Issue by biancadanforth** [mozilla/fathom#142]
As mentioned in #141 , labeling pages currently requires labeling elements using developer tools with a Fathom extension called FathomFox. This labeling involves manual entry of the value for the `data-fathom` attribute for each element to be labeled. When there are possibly upwards of 150 pages to label (#139 ) and multiple elements per sample to label, it's very possible for typos to occur.

Can we reduce or eliminate manual data entry in page labeling?

One idea is to have the user enter each `type` value only once (e.g. `"image"`, `"title"`, `"price"`, ...), select the element in the developer tools' [Inspector panel](https://developer.mozilla.org/en-US/docs/Tools/Page_Inspector), and click a "Tag as ${type}" button that automatically applies the `data-fathom="${type}"` attribute to the opening tag.

Building on that idea, danielhertenstein suggested having the button inside a right-click context menu. Certainly there is a [`browser.menus` WebExtension API](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/API/menus) that may make that possible.

This issue is related to https://github.com/mozilla/fathom-fox/issues/14.

**Comment by biancadanforth** [comment]
Hmm... this is probably a FathomFox issue more than a Fathom issue in terms of where the change will happen, though it's hard to track issues across repos. @erikrose , what would you recommend?

**Comment by biancadanforth** [comment]
Closing this in favor of https://github.com/mozilla/fathom-fox/issues/53 .
